### PR TITLE
Migrate zones components to RTL part 1

### PR DESCRIPTION
**Pull request by ndv99**
## Done

Migrated the following test components to RTL:
- `app/zones/views/ZonesList/ZonesListForm/ZonesList.test.tsx`
- `app/zones/views/ZonesList/ZonesListForm/ZonesListForm.test.tsx`
- `app/zones/views/ZonesList/ZonesListForm/ZonesListHeader.test.tsx`

### CODE REVIEW

File: src/app/zones/views/ZonesList/ZonesListHeader/ZonesListHeader.test.tsx [hunk 1]
```diff
@@ -1,4 +1,6 @@
-import { mount } from "enzyme";
+// import { mount } from "enzyme";
```
Comment (petermakowski) [inline src/app/zones/views/ZonesList/ZonesListHeader/ZonesListHeader.test.tsx]: Looks like you forgot to remove this.

**Review by petermakowski — COMMENTED** [review]
Well done on your first PR! You may need to enable automati linting on save in your editor, looks like prettier has a few things to say. 🙃

**Comment by ndv99** [comment]
I forgot about the other three files 🤦‍♂️ I promise I'll get better at this!

**Review by petermakowski — APPROVED** [review]
LGTM

**Pull request merged**